### ISSUE TITLE
Could not find artifact com.google.protobuf:protoc:exe:osx-x86_64:3.13.0 in nexus

### ISSUE THREAD

**Issue by imranrazakhan** [apache/camel-quarkus#2097]
In my application i am using camel-quarkus-protobuf, but on creating jar i am getting following error.

```
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.10.5.Final:build (build) on project ipc-manager: Failed to build quarkus application: Failed to bootstrap the application: Failed to create the a
pplication model for com.imran.ycube.yq.ipc:ipc-manager::jar:3.0.0-SNAPSHOT: Failed to resolve artifacts: The following artifacts could not be resolved: com.google.protobuf:protoc:exe:osx-x86_64:3.13.0, io.gr
pc:protoc-gen-grpc-java:exe:osx-x86_64:1.32.1: Could not find artifact com.google.protobuf:protoc:exe:osx-x86_64:3.13.0 in nexus (http://nexus.x.x/nexus/content/groups/public)
```
Apache Camel --> 3.7.0
Quarkus --> 1.10.5.Final

I am preparing from windows environment i dont know why its asking for osx.

**Comment by ppalaga** [comment]
Generally, your pom.xml should resemble our [protobuf integration test](https://github.com/apache/camel-quarkus/blob/master/integration-tests/grpc/pom.xml). It is hard to say anything more without a reproducer project.

**Comment by imranrazakhan** [comment]
@ppalaga i will check with your suggestion but for time being i fixed my issue by running following two

```
mvn install:install-file -DgroupId=com.google.protobuf -DartifactId=protoc -Dversion=3.13.0 -Dclassifier=osx-x86_64 -Dpackaging=exe -Dfile=protoc-3.13.0-osx-x86_64.exe
	  
mvn install:install-file -DgroupId=io.grpc -DartifactId=protoc-gen-grpc-java -Dversion=1.32.1 -Dclassifier=osx-x86_64 -Dpackaging=exe -Dfile=protoc-gen-grpc-java-1.32.1-osx-x86_64.exe
	 
```
In my nexus i checked  **protoc-3.13.0-linux-x86_64.exe** and **protoc-3.13.0-windows-x86_64.exe** were available but **protoc-3.13.0-osx-x86_64.exe** was missing, it look like nexus was refusing to upload osx.  i tried to exclude **osx** variant in **camel-quarkus-protobuf** but it didnt work.

**Comment by ppalaga** [comment]
All the platform specific protoc binaries are non-optional deps of `io.quarkus:quarkus-grpc-codegen` that our `camel-quarkus-protobuf-deployment` depends on. I do not think it is possible to exclude by classifier. So installing manually to your local maven repo seems to be the only workaround if you cannot have the artifacts on Nexus.

If you think `io.quarkus:quarkus-grpc-codegen` should be smarter (e.g. have profiles for the individual platforms) about those platform specific dependencies, you may want to file a new issue [under the Quarkus project](https://github.com/quarkusio/quarkus/issues/new/choose)

I am closing, because I do not think we can do anything about this in Camel Quarkus.
